src/clj/tic_tac_toe_clojure/core.clj

```diff
@@ -57,14 +79,41 @@
 (defn game-over? [board]
   (if (or (three-aligned? board) (board-full? board)) true false))
 
-(defn next-player-turn [board user-symbol]
+(defprotocol Player
+  (get-symbol [this])
+  (get-move [this board]))
+
+(deftype HumanPlayer [symbol]
+  Player
+  (get-symbol [this] symbol)
+  (get-move [this board]
+    (get-human-position)))
+
+(deftype ComputerPlayer [symbol]
+  Player
+  (get-symbol [this] symbol)
+  (get-move [this board]
+    (get-first-available-position board)))
+
+(defn next-player-turn [board current-player other-player]
```

Comment:
The name of this function makes sense if there is a next turn, but what if the game is over? Maybe it could be improved in terms of readability? Maybe something like `next-game-status`, or `game-flow`?

Reply:
I'll keep your comment in mind - I'll leave it as is for now, but I'll think about what it does and rename accordingly if needed.